src/baskerville/scripts/hound_transfer.py

```diff
@@ -0,0 +1,389 @@
+#!/usr/bin/env python
+# Copyright 2023 Calico LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========================================================================
+import argparse
+import json
+import os
+import shutil
+import re
+
+import numpy as np
+import pandas as pd
+import tensorflow as tf
+from tensorflow.keras import mixed_precision
+
+from baskerville import dataset
+from baskerville import seqnn
+from baskerville import trainer
+from baskerville import layers
+from baskerville.helpers import transfer_helper
+
+"""
+hound_transfer.py
+
+Modified from hound_train.py.
+Additional argument to allow for transfer learning from existing Hound model.
+"""
+
+def main():
+    parser = argparse.ArgumentParser(description="Train a model.")
+    parser.add_argument(
+        "-k",
+        "--keras_fit",
+        action="store_true",
+        default=False,
+        help="Train with Keras fit method [Default: %(default)s]",
+    )
+    parser.add_argument(
+        "-m",
+        "--mixed_precision",
+        action="store_true",
+        default=False,
+        help="Train with mixed precision [Default: %(default)s]",
+    )
+    parser.add_argument(
+        "-o",
+        "--out_dir",
+        default="train_out",
+        help="Output directory [Default: %(default)s]",
+    )
+    parser.add_argument(
+        "-l",
+        "--log_dir",
+        default="log_out",
+        help="Tensorboard log directory [Default: %(default)s]",
+    )    
+    parser.add_argument(
+        "--restore",
+        default=None,
+        help="Restore model and continue training [Default: %(default)s]",
+    )
+    parser.add_argument(
+        "--trunk",
+        action="store_true",
+        default=False,
+        help="Restore only model trunk [Default: %(default)s]",
+    )
+    parser.add_argument(
+        "--transfer_mode",
+        default="full",
+        help="transfer method. [full, linear, adapter]",
+    )
+    parser.add_argument(
```

Comment:
try to move these script arguments into the params json

Reply:
done.